src/kvstore/raftex/Host.cpp

```diff
@@ -277,6 +291,14 @@ void Host::appendLogsInternal(folly::EventBase* eb,
                         cpp2::AppendLogResponse r;
                         r.set_error_code(res);
                         self->setResponse(r);
+                    } else if (self->lastLogIdSent_ == resp.get_last_log_id()) {
```

Comment:
👍, this block and code block line 229, will prevent from sending logs that follower have received. But still confused about why the leader always trying to seek log 1?